### PR TITLE
Add compute_70 Jenkins test

### PR DESCRIPTION
**Pull request by asi1024**

### CODE REVIEW

File: scripts/ci/chainerx/jenkins/run.sh [hunk 1]
```diff
@@ -75,6 +75,11 @@ case "${CHAINERX_JENKINS_TEST_TYPE}" in
         CHAINERX_NVCC_GENERATE_CODE=arch=compute_50,code=sm_50 MAKEFLAGS=-j16 run_step python_build
         run_step python_test_chainerx
         ;;
+    'chainerx-py3-cc61')
+        run_step setup_conda_environment
+        CHAINERX_NVCC_GENERATE_CODE=arch=compute_61,code=sm_61 MAKEFLAGS=-j16 run_step python_build
```
Comment (niboshi) [inline scripts/ci/chainerx/jenkins/run.sh]: Please write a comment about why 61.

Comment (asi1024) [reply scripts/ci/chainerx/jenkins/run.sh]: Considering the test for TensorCore, should compute capability be >=70?

Comment (asi1024) [reply scripts/ci/chainerx/jenkins/run.sh]: According to the CUDA documentation, Tensor Core requires 70 at least.
https://docs.nvidia.com/cuda/cuda-c-programming-guide/index.html#compute-capabilities

**Comment by asi1024** [comment]
#6843 will resolve this issue.

**Pull request closed without merging**